docusaurus.config.js

```diff
@@ -7,6 +7,9 @@ const docsSiteUrl = process.env.DOCS_SITE_URL || "https://docs.camunda.io";
 const { themes } = require("prism-react-renderer");
 
 module.exports = {
+  future: {
+    experimental_faster: true,
+  },
```

Comment:
WDYT about adding a comment here to the Docusaurus docs about this? You do explain it in the PR.

Reply:
sure, adding!